Automattic/jetpack
# Site Migrations: Add tracks events for AIOWPM start, end, and error events; include blog ID in Tracks events

Related to https://github.com/Automattic/dotcom-forge/issues/7645

## Proposed Changes

* Add Tracks events for the AIOWPM plugin in the site migrations helper functions.
* Ensure we include the blog ID when we're recording tracks events via Jetpack.
* These Tracks events will give us some sense of how many migrations are started, finished, and have errors from the AIOWPM plugin.

Example output:

<img width="736" alt="Screenshot 2024-06-18 at 3 32 21 PM" src="https://github.com/Automattic/jetpack/assets/2124984/6885bdfd-98d5-4d85-bca8-f035eb573e25">

## Does this pull request change what data or activity we track or use?

Yes, it adds the site's blog ID to any Tracks events recorded by `wpcomsh_record_tracks_event`. We need to know if this is allowed according to our TOS.

## Testing instructions:



1. Create a new Business site, mark it as a WoA Dev site, and take it Atomic.
2. Activate this branch on your WoA test site via the Jetpack Beta plugin
3. Rsync the wpcomsh changes to your WoA site: `jetpack rsync wpcomsh yourtestsite.wordpress.com@sftp.wp.com:htdocs/wp-content/mu-plugins`
4. On your WoA test site, install the AIOWPM plugin: https://wordpress.com/plugins/all-in-one-wp-migration/
5. Activate the plugin and go to All in One WP Migration -> Export
6. Create and download an export file
7. Go to All in One WP Migration -> Import
8. Start the import from the file you created
9. Check Tracks Live for the migration start and end events (may take a few minutes to get there): `wpcom_aiowpm_import_start` and `wpcom_aiowpm_import_end`
